subtitles.go

```diff
@@ -927,3 +930,32 @@ func escapeHTML(i string) string {
 func unescapeHTML(i string) string {
 	return htmlUnescaper.Replace(i)
 }
+
+func NewScanner(i io.Reader) *bufio.Scanner {
+	var scanner = bufio.NewScanner(i)
+	scanner.Split(splitLines)
+	return scanner
+}
+
+func splitLines(data []byte, atEOF bool) (advance int, token []byte, err error) {
```

Comment:
I don't really see the point of creating an extra `splitLines` function, since `newScanner` is simple enough could you add the split function directly as an anonymous function:
```go
scanner.Split(func(data []byte, atEOF bool) (advance int, token []byte, err error) {
  ...
})
```

Reply:
Updated.